### DISCUSSION TITLE
Open Vomnibar and select all

### DISCUSSION THREAD

**Discussion by Dim-Tim-1963** [gdh1995/vimium-c#701] [Q&A] [answered]
Already found an answer. :)

**Comment by gdh1995** [comment, marked as answer]
Would you mind sharing your solution?

My idea is a little complex:
1. add this to the end of `Custom search engines`:
```
url: $s re=/^[]*$/ URL
```

2. and add this to `Custom key mappings`:
```
map Fe Vomnibar.activateUrl keyword="url"
```

Then `Fe` will make Vomnibar select the URL part.

**Comment by Dim-Tim-1963** [comment]
Well, it turned out that I just need to open Vomnibar (with "O", or assign another handy hotkey).  Because I want to use it instead of opening new tab and starting type an URL or search.

But it would be great to have a "systematic" introduction to Vimium, because it's a great but rather complicated tool.
